arch/arm64/configs/bcm2711_defconfig

```diff
@@ -1078,6 +1078,7 @@ CONFIG_SND_SOC_SPDIF=m
 CONFIG_SND_SOC_WM8804_I2C=m
 CONFIG_SND_SOC_WM8960=m
 CONFIG_SND_SIMPLE_CARD=m
+CONFIG_SND_DACBERRY400=m
```

Comment:
Is this really only for Pi 4s?

Reply:
No, it is also for Pi 3. Added in Pi 3 related config files.